File: src/test/java/uk/co/crunch/impl/v2x/AlertRulesGenerator2xTest.java
package uk.co.crunch.impl.v2x;

import org.junit.Test;

import static org.assertj.core.api.Assertions.assertThat;
import static uk.co.crunch.impl.v2x.AlertRulesGenerator2x.titlecase;

public class AlertRulesGenerator2xTest {

    @Test
    public void testTitleCase() {
        assertThat(INSTANCE.titlecase("")).isEqualTo("");
        assertThat(INSTANCE.titlecase("audit-service")).isEqualTo("AuditService");
        assertThat(INSTANCE.titlecase("audit_service")).isEqualTo("AuditService");
        assertThat(INSTANCE.titlecase("audit.service")).isEqualTo("AuditService");
    }
}

File: src/test/java/uk/co/crunch/impl/v2x/AlertRulesGenerator2xTest.kt
package uk.co.crunch.impl.v2x

import org.assertj.core.api.Assertions.assertThat
import org.junit.Test
import uk.co.crunch.impl.v2x.AlertRulesGenerator2x.titlecase

class AlertRulesGenerator2xTest {

    @Test
    fun testTitleCase() {
        assertThat(titlecase("")).isEqualTo("")
        assertThat(titlecase("audit-service")).isEqualTo("AuditService")
        assertThat(titlecase("audit_service")).isEqualTo("AuditService")
        assertThat(titlecase("audit.service")).isEqualTo("AuditService")
    }
}